src/NuGet.Core/NuGet.Packaging/PackagingConstants.cs

```diff
@@ -44,6 +44,18 @@ public static class Folders
                 Analyzers,
                 Source
             };
+
+            public static string[] SupportFrameworks { get; } = new string[] {
```

Comment:
Can we make this `internal`? It doesn't seem to be used outside of `NuGet.Packaging`, and I don't think it's particularly useful for customers using this package.